alivee/utils/requests/request_meta.py

```diff
@@ -0,0 +1,62 @@
+import urllib
+
+from utils.requests.exceptions import RequestMethodNotSupportedException
+
+
+class RequestMeta(object):
+
+    def __init__(self,
+                 host,
+                 path='',
+                 params=None,
+                 method=None,
+                 headers=None,
+                 body=None,
+                 protocol='https'):
```

Comment:
Curious why making this multiple lines, maybe we can sycn next time for this style